src/app/modules/main/chat_section/module.nim

```diff
@@ -147,7 +147,12 @@ proc buildChatSectionUI(
       colorHash = self.controller.getColorHash(chatDto.id)
       colorId = self.controller.getColorId(chatDto.id)
 
-    let amIChatAdmin = (self.amIMarkedAsAdminUser(chatDto.members) or channelGroup.admin)
+    # for group chats only member.admin should be checked,
+    # because channelGroup.admin is alway true
```

Comment:
this might be a bug on status-go

Reply:
Hmm. Channel group contains a lot of chats. There can be 1:1 chats, group chats, global chats. This parameter makes sense only for 1:1 chat and community groups.
Maybe the better approach would be to move admin flag to chat structure rather then having it for a group.